packages/taco/integration-test/sign.test.ts

```diff
@@ -98,6 +100,76 @@ describe.skipIf(!process.env.RUNNING_IN_CI)(
       );
       expect(signResult.signingResults).toBeDefined();
       expect(Object.keys(signResult.signingResults).length).toBeGreaterThan(0);
-    }, 15000);
+    }, 150000);
```

Comment:
This was fixed before, perhaps undone as part of your rebase?

In any case, it should be 15s and not 150s.

```suggestion
    }, 15000);
```